packages/ketcher-core/src/application/editor/actions/closelyFusing.ts

```diff
@@ -98,3 +106,52 @@ function closestToMerge(struct, closestMap) {
 
   return mergeMap
 }
+
+function mergeItemsFuseMap(items: Map<number, number>): Set<number> {
```

Comment:
This function name is not clear to me.
Does this function filter out duplicated items?
Then maybe filterOutDuplicatedItems?
Thoughts?

Reply:
Well, it merges items into one array. Thus — mergeItems.
I can name it "mergeItemsIntoOneArrayFuseMap" if it helps.
Can't get rid of "FuseMap" part because it's highly niсhe and not supposed to be used anywhere else.

Reply:
can't say 'mergeItemsIntoOneSetFuseMap' looks easier to understand from my point of view 🤔 
Can you please explain what this function does if it doesn't filter out duplicated items?
And why is this 'fuseMap' thing needed? What does it mean?

Reply:
> Can you please explain what this function does if it doesn't filter out duplicated items?
->
> ...it merges items into one Set.

It sure does remove any duplicated items, but it's not the point of this function.

> And why is this 'fuseMap' thing needed?
->
> ...because it's highly niсhe and not supposed to be used anywhere else.

The wording of this part can be tweaked, just not removed entirely, so the whole sense would not be lost.

Reply:
> It sure does remove any duplicated items, but it's not the point of this function.
> The wording of this part can be tweaked, just not removed entirely, so the whole sense would not be lost.

What is the point of this function then? Sorry, I struggle to grasp the whole sense of this function 🤷

Reply:
Clarified in direct messages. 
And renamed a little to make the process more transparent.